src/main/build/debug.c

```diff
@@ -60,6 +60,7 @@ const char * const debugModeNames[DEBUG_COUNT] = {
     DEBUG_NAME(MAX7456_SPICLOCK),
     DEBUG_NAME(SBUS),
     DEBUG_NAME(FPORT),
+    DEBUG_NAME(FPORT2),    
```

Comment:
New debug items _MUST_ go to the end of the list

Reply:
Done